platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/junit/OtterLogTestExtension.java

```diff
@@ -123,11 +123,7 @@ private static void clear(@NonNull final ExtensionContext extensionContext) {
         final TestEnvironment testEnvironment =
                 (TestEnvironment) extensionContext.getStore(EXTENSION_NAMESPACE).remove(ENVIRONMENT_KEY);
         if (testEnvironment != null) {
-            try {
-                testEnvironment.destroy();
-            } catch (InterruptedException e) {
-                throw new RuntimeException(e);
-            }
+            testEnvironment.stop();
```

Comment:
Why is this change needed? We want a new `TestEnvironment` for each test. Destroying it at the end of a test seems to be the right thing to do.

Reply:
Fixed